care/emr/resources/tag/config_spec.py

```
@@ -0,0 +1,151 @@
+    drug = "drug"
+    lab = "lab"
+    admin = "admin"
+    contact = "contact"
+    clinical = "clinical"
+    behavioral = "behavioral"
+    research = "research"
+    advance_directive = "advance_directive"
+    safety = "safety"
+
+
+class TagResource(str, Enum):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)